pkg/version/version.go

```diff
@@ -59,6 +60,9 @@ func NewVersionCommand(basename string) *cobra.Command {
 		Run: func(c *cobra.Command, args []string) {
 			fmt.Printf("%s %v\n", basename, Get())
 			fmt.Printf("kubernetes %v\n", kubeversion.Get())
+			if basename == "openshift" {
```

Comment:
This doesn't work - you have to support atomic-enterprise, origin, others.  You should pass a boolean in that says "print etcd" and set that in the caller.